CIFAR10/train/fastai_pytorch.json

```diff
@@ -0,0 +1,17 @@
+{
+    "version": "v1.0",
+    "author": "fast.ai + students",
+    "authorEmail": "j@fast.ai",
+    "framework": "fastai / pytorch",
+    "codeURL": "https://github.com/fastai/imagenet-fast/commit/2748de3b0e28ecca1e02d90b2d5bf7d9b9e58e3d",
+    "model": "Custom Wide Resnet (https://github.com/fastai/imagenet-fast/blob/2748de3b0e28ecca1e02d90b2d5bf7d9b9e58e3d/cifar10/models/wideresnet.py#L60)",
```

Comment:
@brettkoonce can you more the model link to `misc`? I don't think this will display well on the leaderboard.

Reply:
done!

Reply:
Sorry I should have clarified. Change the `model` field to "Custom Wide ResNet" and move the url to `misc` maybe with a field called `modelURL`